bin/jsdoc-to-ts.js

```diff
@@ -47,7 +50,7 @@ if (opts.help) displayHelp();
 jsdocToTs({
 	package: opts._[0] || '.',
 	output: fs.writeFileSync,
-	ignore: ['node_modules', 'ilib', 'build', 'dist', 'samples', 'coverage', 'tests'],
+	ignore: ['node_modules', 'ilib', 'build', 'dist', 'samples', 'coverage', 'tests', 'docs'],
```

Comment:
Why docs is added?

Reply:
I will remove the 'docs' and re-check on another ticket.